### PR TITLE
feat(example-webpack): add an example to generate bundles with webpack

### PR DESCRIPTION
**Pull request by raymondfeng**
Running LoopBack's Core inside a browser!

Relates to https://github.com/strongloop/loopback-next/issues/2366

## Checklist

- [x] DCO (Developer Certificate of Origin) [signed in all commits](https://loopback.io/doc/en/contrib/code-contrib.html)
- [x] `npm test` passes on your machine
- [x] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

**Comment by bajtos** [comment]
Nice!

How are we going to test that the bundled version actually works in real browsers? We should run such tests as part of our CI build, preferably only when example dependencies change.

**Comment by raymondfeng** [comment]
> How are we going to test that the bundled version actually works in real browsers? We should run such tests as part of our CI build, preferably only when example dependencies change.

I added an integration using [zombie](http://zombie.js.org/) to test against browsers.

**Comment by raymondfeng** [comment]
`Coverage decreased (-15.5%) to 75.843%`? 😮 

It's very strange that the newly added `example-webpack` now has 100% coverage. How could it decreases the overall percentage by 15.5%?

**Comment by raymondfeng** [comment]
The example can be previewed at https://jsfiddle.net/4t5drpq7/4/. I published it to npm under `loopback4-example-webpack` for testing. The official npm package will be `@loopback/example-webpack` when it's released.

**Comment by raymondfeng** [comment]
@bajtos PTAL

**Review by jannyHou — APPROVED** [review]
👍 Tested the app. When refreshing the browser the timestamp reflects the current time.

File: packages/cli/generators/example/index.js [hunk 1]
```diff
@@ -57,6 +57,7 @@ const EXAMPLES = {
     'An example implmenting authentication in a LoopBack application using Passport modules.',
   ),
   'todo-jwt': g.f('A modified Todo application with JWT authentication'),
+  webpack: g.f('An example to bundle @loopback/core using webpack'),
```
Comment (dhmlau) [inline packages/cli/generators/example/index.js:60]: I think it needs the quote. 
```suggestion
  'webpack': g.f('An example to bundle @loopback/core using webpack'),
```

**Review by dhmlau — APPROVED** [review]
Based on my limited understanding of what this example does, it LGTM. One comment about the example/index.js change.

File: packages/cli/generators/example/index.js (hunk 1 above)
Comment (raymondfeng) [reply packages/cli/generators/example/index.js:60]: No. `prettier` will remove `'` if the key is a valid js token.

**Pull request merged**